code/datums/explosion.dm

```diff
@@ -69,6 +69,12 @@ GLOBAL_LIST_EMPTY(explosions)
 		flash_range = min(GLOB.MAX_EX_FLASH_RANGE * cap_multiplier, flash_range)
 		flame_range = min(GLOB.MAX_EX_FLAME_RANGE * cap_multiplier, flame_range)
 
+	for(var/obj/effect/explosionblocker/E in world)
```

Comment:
Undocumented change

Reply:
it's in the changelog

tweak: explosives no longer work on away missions

same reason for the bomb maxcap being 0 in away missions.